### ISSUE TITLE
Add (configurable) app-wide locking for Cryptomator

### ISSUE THREAD

**Issue by markuskreusch** [cryptomator/cryptomator-ios#58]
Users should be able to configure a timeout that when expired triggers locking of the application. This will prevent access to the vaults and especially cloud services through the Cryptomator application.

**Comment by yosephyoo** [comment]
I agree! Allowing access to vaults (renaming/moving/deleting) and editing settings (auto-lock interval) poses a security risk to users.
